packages/elements/package.json

```diff
@@ -53,6 +53,7 @@
     "mutation-testing-report-schema": "2.0.3",
     "postcss": "8.4.31",
     "postcss-loader": "7.3.3",
+    "prism-svelte": "^0.5.0",
```

Comment:
This package hasn't been updated in about 2 years. I'm not sure this is the way to go. Maybe we can get away with "good-enough" formatting by using html for svelte?

Reply:
Yes, I thought of that. However:

- `prism-svelte` does have 20k downloads / week. 
- Svelte syntax is pretty stable. So it might be a stable package because of that.
- I see some usage 'in the wild'. Some examples:
    - https://magidoc.js.org/svelte-plugins/prismjs
    - https://lihautan.com/notes/svelte-with-prism/#with-prism-svelte
- C'mon man, this looks awesome 🤩
     ![image](https://github.com/stryker-mutator/mutation-testing-elements/assets/1828233/924476ec-a7c9-4095-9f96-18d877ea1526)

Reply:
vs screenshot of html formatting: 
![image](https://github.com/stryker-mutator/mutation-testing-elements/assets/1828233/f9aeed41-5a01-42b6-ad95-c43d8d8e4db1)

Reply:
It looks nice!

I don’t think there’s a better alternative (though I must admit, I didn’t search for that long). Personally I am okay with this 👌

Reply:
are you ok with this @hugo-vrijswijk ?

Reply:
Yeah it does look nice 😁